packages/ember-application/lib/system/application.js

```diff
@@ -645,24 +795,58 @@ var Application = Namespace.extend(RegistryProxy, {
     @method didBecomeReady
   */
   didBecomeReady() {
-    if (this.autoboot) {
-      this.runInstanceInitializers(this.__deprecatedInstance__);
-
-      if (environment.hasDOM) {
-        this.__deprecatedInstance__.setupEventDispatcher();
-      }
-
-      this.ready(); // user hook
-      this.__deprecatedInstance__.startRouting();
-
+    try {
+      // TODO: Is this still needed for _globalsMode = false?
       if (!Ember.testing) {
```

Comment:
I don't remember offhand where `BOOTED` is used, but I'm pretty sure we use the container key to generate `toString()` names of objects in Ember CLI across the board and so could skip `Ember.Namespace.processAll()`.